shrink station selector items

PatrickCleary opened the pull request:
## Motivation
In v3 you can see all the stations immediately on dropdown. New station selection items were too big and the container was too small.

I also think we should remove handicap and bike icons, as that is not really the purpose of our app. I think it would be more useful to use that space for branches (A/B)

## Changes
**Before**
<img width="1417" alt="Screenshot 2023-07-01 at 10 45 49 AM" src="https://github.com/transitmatters/t-performance-dash/assets/46229349/3ff2e9f4-0e43-42f8-b736-a3c9319dd849">

**after**
<img width="1428" alt="Screenshot 2023-07-01 at 10 45 33 AM" src="https://github.com/transitmatters/t-performance-dash/assets/46229349/3bab7c3f-f332-4f5c-b7ae-fc05182f35b9">

<!-- What does this change exactly? Include relevant screenshots, videos, links -->

## Testing Instructions

<!-- How can the reviewer confirm these changes do what you say they do? -->

devinmatte commented:
I feel strongly about keeping the bike and accessibility icons

devinmatte commented:
My main concern with shrinking them is a touch radius on mobile. Want to make sure it's not too easy to misclick

PatrickCleary commented:
**Update**

Reverted size of mobile items.

Added branch info

Fixed bug where if `to` station is branch `D` and a `from` station is selected on another branch, the app breaks. Now the `to` station will be auto-updated to a core station (gov center for green, park for red).,
